spec.md

```diff
@@ -602,6 +602,12 @@ This RPC will be called by the CO to provision a new volume on behalf of a user
 This operation MUST be idempotent.
 If a volume corresponding to the specified volume `name` already exists and is compatible with the specified `capacity_range`, `volume_capabilities` and `parameters` in the `CreateVolumeRequest`, the Plugin MUST reply `0 OK` with the corresponding `CreateVolumeResponse`.
 
+Plugins MAY create 3 types of volumes:
+
+- Empty volumes. When plugin supports `CREATE_DELETE_VOLUME` capability.
+- From an existing snapshot. When plugin supports `CREATE_DELETE_SNAPSHOT` optional capability .
+- From an existing volume. When plugin supports efficient cloning, and reports the optional capability `CLONE_VOLUME`.
```

Comment:
This seems like an implementation detail. Should we support naive copies and leave it up to the plugin to handle it?

Reply:
I wanted to indicate that the idea is for efficient cloning.  Not an attach vol1, attach vol2, then use dd.
Plugins can ignore it and internally do that, that's why I didn't add any MUST or REQUIRED. 
But plugins using the host assisted cloning mechanism (dd)  would give terrible performance and in my opinion would be frustrating.
Do you think I should remove the "efficient" adjective?